docs/src/app/api/internal/[transport]/route.ts

```
@@ -104,6 +133,7 @@ const handler = createMcpHandler(
     basePath: "/api/internal",
     verboseLogs: true,
     maxDuration: 120,
```

Biome (2.4.10)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)